tfhe/src/integer/gpu/list_compression/server_keys.rs

```diff
@@ -263,6 +263,10 @@ impl CudaDecompressionKey {
 
         match &self.blind_rotate_key {
             CudaBootstrappingKey::Classic(bsk) => {
+                assert!(
```

Comment:
Why don't we need noise reduction in decompression? And isn't it needed for compression?